src/utils/builder.ts

```diff
@@ -20,26 +29,131 @@ export function createNewPathway(name: string, description?: string, pathwayId?:
 }
 
 export function exportPathway(pathway: Pathway): string {
+  const elm: ElmLibrary = {
+    library: {
+      identifier: {
+        id: pathway.id,
+        version: '1.0.0'
+      },
+      schemaIdentifier: {
+        id: 'urn:hl7-org:elm',
+        version: 'r1'
+      },
+      usings: {
+        def: [
+          {
+            localIdentifier: 'System',
+            uri: 'urn:hl7-org:elm-types:r1'
+          },
+          {
+            localId: '1',
+            locator: '3:1-3:26',
+            localIdentifier: 'FHIR',
+            uri: 'http://hl7.org/fhir',
+            version: '4.0.0'
+          }
+        ]
+      },
+      statements: {
+        def: [
+          {
+            locator: '13:1-13:15',
+            name: 'Patient',
+            context: 'Patient',
+            expression: {
+              type: 'SingletonFrom',
+              operand: {
+                locator: '13:1-13:15',
+                dataType: '{http://hl7.org/fhir}Patient',
+                type: 'Retrieve'
+              }
+            }
+          }
+        ]
+      },
+      includes: { def: [] },
+      valueSets: { def: [] }
+    }
+  };
+
+  const mergeElm = (additionalElm: ElmLibrary): void => {
```

Comment:
I think it would be better to define this as another function alongside its encapsulating method `exportPathway` and just pass `elm` in as an argument.

Although the closure is nice this method is going to be redefined everytime it runs, `exportPathway` is already >100 lines long, and it only has a single argument as is.  Plus, I could see this method being useful elsewhere in the future.